app/models/grda_warehouse/enrollment_change_history.rb

```diff
@@ -6,17 +6,30 @@
 
 module GrdaWarehouse
   class EnrollmentChangeHistory < GrdaWarehouseBase
-    belongs_to :client
+    belongs_to :client, class_name: 'GrdaWarehouse::Hud::Client'
     validates_presence_of :on, :client_id
 
+    RETENTION_DURATION = 6.months
+    scope :expired_as_of, ->(date) {
+      expiration_date = date - RETENTION_DURATION
+      where(on: ...expiration_date)
```

Comment:
use "on" rather than "created_at" - it seems like "on" is just the date the job was run (attributes_for_client_on_date)